analytics/cdk/emr-eks/bin/emr-eks.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import EmrEksStack from '../lib/emr-eks-stack';
+import { Stack } from 'aws-cdk-lib';
+import { EmrEksTeamProps } from '../lib/teams/emrEksTeam';
+import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+const app = new cdk.App();
+
+const account = '123456789012';
```

Comment:
It may be better to get account and region from the profile/session instead of hardcoding in the pattern. It's one less thing for the user to modify in the code before deploying.